addons/website_sale/data/demo.xml

```diff
@@ -268,18 +258,17 @@
             <field name="public_categ_ids" eval="[(6,0,[ref('public_category_lamps')])]"/>
         </record>
 
-        <record model="product.pricelist" id="product.list0">
-            <field name="selectable" eval="True" />
-            <field name="sequence">3</field>
-            <field name="currency_id" ref="base.USD"/>
-        </record>
-
         <record id="benelux" model="res.country.group">
             <field name="name">BeNeLux</field>
             <field name="country_ids" eval="[(6,0,[
                 ref('base.be'),ref('base.lu'),ref('base.nl')])]"/>
         </record>
 
+        <!-- Since we are adding pricelists, we activate the feature -->
+        <record id="base.group_user" model="res.groups">
```

Comment:
Should not be done in a `noupdate` block.

Reply:
It's only a demo data file, do we really care ?

Reply:
No; so, let's keep this data where it is so it's contextually close to the other pricelist data.